integrations/okteto/manifest.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Okteto",
+  "version": "1.0.0",
+  "overview": "Automatically create a LaunchDarkly environment for every development or preview environment.",
+  "description": "The LaunchDarkly Okteto integration enables you to automatically create a LaunchDarkly environment every time your team creates a development or preview environment. With this, your team will be able to use feature flags as part of the development lifecycle. Okteto will automatically manage the lifecycle of the LaunchDarkly environment for you. When an environment is created in Okteto, the LaunchDarkly environment will be created automatically, and when the environment is destroyed, the LaunchDarkly environment will be destroyed as well.",
```

```suggestion
  "description": "Automatically create a LaunchDarkly environment every time your team creates a development or preview environment in Okteto. This integration ensures your new LaunchDarkly environment always has the most up-to-date flag targeting. Okteto will automatically manage the lifecycle of the LaunchDarkly environment for you.",
```

We don't currently have support for Markdown so suggesting some slightly different wording so it's easier to parse in the format.